src/main/scala/com/cognite/sdk/scala/v1/resources/dataPoints.scala

```diff
@@ -29,6 +29,7 @@ import io.circe.parser.decode
 import scala.concurrent.duration._
 import scala.util.control.NonFatal
 
+// scalastyle:off number.of.methods
```

Comment:
Seems a bunch of the parse stuff could be moved to a companion object, and we might not need this?